assets/_sass/_navbar.scss

```diff
@@ -20,40 +20,33 @@ $nav-item-border: darken($navbar-default-bg, 8%);
     darken($color, 4%)
   );
   filter: none;
-  border: 1px solid darken($color, 10%);
```

Comment:
Why did you remove the border ? There is no longer a border on the navbar component any longer, is this intentional ?